src/assets/scripts/client/aircraft/AircraftController.js

```diff
@@ -66,6 +69,16 @@ export default class AircraftController {
         this._navigationLibrary = navigationLibrary;
 
         /**
+         * Local reference to static `EventBus` class
+         *
+         * @property _eventBus
+         * @type {EventBus}
+         * @default EventBus
+         * @private
+         */
+        this._eventBus = EventBus;
```

Comment:
thats just a style thing.  it adds a little more context to what is happening.  it is _`this`_ that is triggering or on-ing.